src/pages/Cart/CartPage/Delivery.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import PriceInfo from '../CartComponent/PriceInfo';
+import CheckedCartList from '../CartComponent/CheckedCartList';
+import './Delivery.scss';
+
+class Delivery extends Component {
+  render() {
+    const { cartListData } = this.props;
+
+    return (
+      <div className="Delivery">
+        <div className="DeliveryInfo">
+          <div className="firstBox">
+            <div div className="title">
+              배송
+            </div>
+            <div className="firstBoxFirst">
+              <div className="firstBoxFirstTitle">당일 배송 • 4,000원</div>
+              <input type="radio"></input>
+              <p>
+                당일배송은 오전 00시 01분부터 오후 12시까지 결제 완료한 서울
+                지역 주문에 한하여 진행됩니다. 오후 12시 이후 및 주말/공휴일
+                주문의 경우 돌아오는 첫 영업일 기준으로 순차적으로 배송되며,
+                빠른 배송진행을 위해 당일배송 신청 후, 주소변경 및 주문취소는
+                불가합니다.
+              </p>
+            </div>
+            <div className="firstBoxSecond">
+              <div className="firstBoxSecondTitle">일반 배송 • 무료</div>
+              <input type="radio"></input>
+              <p>
+                입금 확인 후 3~4일 이내(공휴일 제외)에 배달됩니다 (나의 라코스테
+                폴로 등 일부 품목 제외) 그러나 배송기간 중 휴일이 포함되어
+                있거나 제품의 재고 상황에 따라 기간이 다소 지연될 수도 있습니다.
+              </p>
+            </div>
+          </div>
+          <div className="SecondBox">
+            <div div className="title">
+              새 배송지(*필수)
+            </div>
+            <div className="userGender">
+              <div>성별</div>
+              <div className="center">
+                <input type="radio"></input>남성
+              </div>
+              <div>
+                <input type="radio"></input>여성
+              </div>
+            </div>
+            <div>
+              <input type="text" placeholder="이름"></input>
+            </div>
+            <div>
+              <input type="text" placeholder="우편번호"></input>
+            </div>
+            <div>
+              <input type="text" placeholder="주소"></input>
+            </div>
+            <div>
+              <input type="text" placeholder="상세주소"></input>
+            </div>
+            <div>
+              <input type="text" placeholder="나라"></input>
+            </div>
+            <div>
+              <input type="text" placeholder="연락처"></input>
+            </div>
```

Comment:
여기도 Array.map으로 표현 가능해보입니다

Reply:
해당 input에 placeholder 외에 각 input을 구별하는 name이라는 값이 들어가는데 2개의 값을 동시에 map 함수로 적용할 수 있는 방법이 있는지 궁금합니다.

Reply:
```suggestion
            {[{ name: "a", placeholder: "이름" }, ...].map((prop) => (
              <div>
                <input type="text" {...prop} />
              </div>            
            ))}
```
위와 같이 배열 안에 객체가 나열되는 형태로 구성하시고, 배열 안에 들어가는 데이터는 별도로 분리해주시면 되겠네요